doc/libpmem/pmem_is_pmem.3.md

```diff
@@ -103,7 +103,9 @@ following file creation flags:
 
 + **PMEM_FILE_CREATE** - Create the file named *path* if it does not exist.
   *len* must be non-zero and specifies the size of the file to be created.
-  The file is fully allocated to the size *len* using **posix_fallocate**(3).
+  If the file already exists, it will be extended or truncated to *len.*
```

Comment:
I think we should never truncate files passed to pmem_map_file.

Reply:
I agree, we should probably just return a smaller mapping.
Or... we do nothing, which would let users create large mapping that have no file backing (like we do in set.c)